package.json

```diff
@@ -2,25 +2,35 @@
   "name": "react-native-device-info",
   "version": "3.1.1",
   "description": "Get device information using react-native",
-  "main": "deviceinfo.js",
-  "typings": "./deviceinfo.d.ts",
+  "main": "lib/commonjs/index.js",
+  "module": "lib/module/index.js",
+  "react-native": "src/index.js",
+  "types": "lib/typescript/src/index.d.ts",
+  "files": [
+    "/android",
+    "!/android/build",
+    "/ios",
+    "/windows",
+    "/web",
+    "src/",
+    "lib/",
+    "/*.podspec"
+  ],
   "repository": {
     "type": "git",
     "url": "https://github.com/react-native-community/react-native-device-info"
   },
   "scripts": {
-    "analyze": "yarn ts-check && yarn flow-check",
-    "dev-sync": "cp -r deviceinfo* *podspec windows web android ios example/node_modules/react-native-device-info/",
-    "flow-check": "npx flow check-contents < deviceinfo.js.flow",
+    "analyze": "npx tsc --noEmit",
+    "dev-sync": "cp -r deviceinfo* *podspec windows web android ios src example/node_modules/react-native-device-info/",
     "lint": "npx eslint ./ --ignore-pattern example --ignore-pattern node_modules --fix --quiet",
-    "pretty": "prettier --write deviceinfo.js",
     "shipit": "np",
-    "ts-check": "npx tsc deviceinfo.d.ts --noEmit",
     "test": "echo no tests defined",
-    "precommit": "lint-staged && yarn analyze"
+    "precommit": "lint-staged && yarn analyze",
+    "prepare": "bob build"
```

Comment:
is "prepare" a community standard for this? I'm always looking for a "build" target personally, I'd think twice if I was coming into this as a fresh potential-contributor just trying a patch really quickly before working more deeply

Reply:
`prepare` gets triggered automatically when you create an npm package to be uploaded.  This should mean that `shipit` will also use it when it runs.

Reply:
Ah - so it is a standard (standard hook at least). That makes sense. This might be pedantic then, but could you add a build target and have prepare call it? I think it will just avoid doubt and if it chains to one underlying command should not increase maintainence